ngraph/test/backend/dft.in.cpp

```diff
@@ -0,0 +1,362 @@
+        0.82389224,   1.8939058,     6.882805,     6.4072084,   -1.3024135,   -0.22483894,
+        -0.22082287,  1.0370905,     -0.7639439,   0.6950346,   -0.731326,    0.16821115,
+        0.0887468,    -0.5732441,    -0.40715322,  -0.96244293, -0.89126545,  1.3140129,
+        -0.42358512,  1.7674587,     -0.6400819,   -1.6113993,  4.4106574,    5.706909,
+        -1.1110737,   0.10560027,    -1.1108764,   0.34190884,  2.1167603,    -0.067495525,
+        -0.16237324,  0.2604496,     -0.8129095,   -0.42274237, -1.1412699,   -0.0011268258,
+        -0.63462454,  -0.15172139,   -0.7164279,   0.14801888,  -0.3538928,   1.583736,
+        4.9876184,    4.2879796,     -0.8491325,   0.5345522,   -0.60507995,  -0.9020085,
+        1.0447598,    0.21135187,    -0.4787205,   -0.3230412,  0.8076494,    -0.04361339,
+        0.62797767,   0.15487206,    -0.23772183,  0.69546384,  1.8609382,    -1.7030516,
+        1.2658813,    -0.6791475,    4.921037,     4.8929176,   -0.0124401,   -0.6873918,
+        -0.21879943,  -0.48610657,   0.36776963,   0.12423802,  -0.7854952,   0.48838156,
+        -0.5085067,   -0.08865434,   1.1653454,    0.81965554,  -0.6399579,   -1.0967884,
+        1.4099771,    -0.15370974,   2.8824244,    1.0534087,   4.7045717,    5.2045445,
+        -0.6350576,   2.5321684,     0.6987691,    -0.53839976, -0.09889791,  0.5662097,
+        0.4088725,    0.635128,      -1.763303,    -0.49720347, -1.0772469,   1.2422445,
+        -0.3619956,   -1.311133,     1.5846866,    1.0530244,   -0.61141044,  0.74831486,
+        5.433625,     3.9661994,     2.006918,     -0.8703619,  -0.7658511,   0.0811044,
+        0.83877516,   -0.63553256,   -0.67563355,  1.7368636,   0.9372277,    1.8246815,
+        0.8615329,    -0.18161502,   0.62479717,   0.2028623,   0.159001,     1.860977,
+        0.04177074,   -0.49050322,   4.9402246,    4.0296063,   -0.74729615,  -0.27802998,
+        -0.8077982,   -0.5414143,    0.467114,     0.9016136,   2.1971147,    -1.466963,
+        -1.2350414,   1.0967304,     -0.95607626,  0.51462483,  0.28838068,   1.0117096,
+        -0.21846394,  0.114624545,   -1.627146,    -0.9431294};
+
+    auto dft_output = backend->create_tensor(element::f32, Shape{2, 10, 10, 2});
+
+    auto backend_data = backend->create_tensor(element::f32, Shape{2, 10, 10, 2});
+    copy_data(backend_data, input_data);
+
+    auto handle = backend->compile(f);
+
+    handle->call({dft_output}, {backend_data});
+
+    auto result = read_vector<float>(dft_output);
+    size_t num_of_elems = result.size();
+    for (std::size_t j = 0; j < num_of_elems; ++j)
+    {
+        EXPECT_NEAR(result[j], expected_dft1d_results[j], 0.00001);
+    }
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, dft2d_eval)
+{
+    auto data = std::make_shared<op::Parameter>(element::f32, Shape{2, 10, 10, 2});
+    auto axes_input = op::Constant::create<int64_t>(element::i64, Shape{2}, {1, 2});
+    auto dft = std::make_shared<op::v7::DFT>(data, axes_input);
```

Comment:
Is there a way to make test for signal_size parameter?

Reply:
It will be done in the separate PR.